README.md

```diff
@@ -15,7 +15,7 @@ Build things 🎉
 
 ## Contributing
 
-[Please check the contribution docs](https://github.com/twilio-labs/paste/blob/master/CONTRIBUTING.md)
+Before contributing, please make sure that you read our [Contributing Guidelines](/CONTRIBUTING.md) and agree with our [Code of Conduct.](/CODE_OF_CONDUCT.md)
```

Comment:
I don't think we should remove the link to our contribution guide.  Just add this underneath.  

```
## Code of Conduct

Please be aware that this project has a [Code of Conduct](https://github.com/twilio-labs/.github/blob/master/CODE_OF_CONDUCT.md). The tl;dr is to just be excellent to each other ❤️
```